app/lab/package.json

```diff
@@ -178,7 +176,6 @@
   "jupyterlab": {
     "extensions": [
       "@jupyterlite/application-extension",
-      "@jupyterlite/theme-extension",
```

Comment:
does `@jupyterlab/theme-extension` need to added to this list? the singleton list?

Reply:
Or `@jupyterlab/apputils`? Since the upstream plugin comes from:

https://github.com/jupyterlab/jupyterlab/blob/14abba086b5cae2ad0bcfaf0f5e08a7dc1ca5678/packages/apputils-extension/src/themesplugins.ts#L37

Reply:
hey, it builds! just looking at diffs